mungegithub/webhooks-publisher/README.md

```diff
@@ -14,3 +17,32 @@ kubectl create configmap webhook-config-v10 --from-file=config.yaml
 
 kubectl apply -f deployment.yaml
 ```
+
+How to use
+----------
+
+`config.yaml` contains the configuration:
+
+- `project` is the project that has the PubSub queue
```

Comment:
Is this the GCE Project Id ?
Is it true that your client must have GCE privileges to publish to that project ? Where are those credential ?